### DISCUSSION TITLE
Home Assistant "sensor.gasmeter_rate_per_time_unit"

### DISCUSSION THREAD

**Discussion by MrReSc** [jomjol/AI-on-the-edge-device#2105] [General]
The sensor "sensor.gasmeter_rate_per_time_unit" has no "state_class: measurement" attribute. This is the flow rate, if I understand correctly. Without this attribute the value is not saved in the long term statistics. Is this attribute missing or is there a reason?

![image](https://user-images.githubusercontent.com/24952268/222051659-c9955f6c-2d37-4272-8c84-8cb50214f357.png)

**Comment by caco3** [comment]
I guess that is a bug. I will correct it.

**Reply by caco3** [reply]
https://github.com/jomjol/AI-on-the-edge-device/issues/2115
